docs/api-reference/deckgl.md

```diff
@@ -75,9 +75,13 @@ This is helpful when rendered objects are difficult to target, for example
 irregularly shaped icons, small moving circles or interaction by touch.
 Default `0`.
 
-##### `pixelRatio` (Number, optional)
+##### `useDevicePixelRatio` (Boolean, optional)
 
-Will use device ratio by default.
+Default value is false.
+
+When true, device's full resolution will be used for rendering.
+
+Note: Only effects devices with retina/HD displays. Set it false (default) unless it is required, as it affects your application performance.
```

Comment:
Add: "DPR can also change when zooming in the browser and when moving windows between screens."?

For the remarks section?